llvm/test/SYCLLowerIR/ConditionalCallOnDevice/conditional-call-on-device-1.ll

```diff
@@ -0,0 +1,37 @@
+; RUN: opt -passes=sycl-conditional-call-on-device < %s -S | FileCheck %s
```

Comment:
I don't see any `CHECK-NOT` or `--implicit-check-not` to ensure that we don't have calls to an old function and that it is not present in a module anymore